backend/linux_input.c

```diff
@@ -170,6 +260,12 @@ void *twin_linux_input_create(twin_screen_t *screen)
     twin_set_file(dummy, tm->fd, TWIN_READ, tm);
 #endif
 
+    /* Start event handling thread */
+    if (pthread_create(&tm->udev_thread, NULL, twin_linux_udev_thread, tm)) {
```

Comment:
Why do you create yet another thread for event handling? Can you deal with udev in the existing thread?

Reply:
Excellent. You have discovered how effective I/O multiplexing can be for optimizing the code.

Reply:
We indeed require only one thread, the `udev` and `evdev` are now handled by one `pollfd` array.